index.html

```diff
@@ -94,174 +94,7 @@ <h1>
-              </p>
-              <ul class="work-stack">
-                <li class="stack-item">html</li>
-                <li class="stack-item">bootstrap</li>
-                <li class="stack-item">Ruby</li>
-              </ul>
-              <a href="" class="btn-a">See Project</a>
-            </article>
-            <article class="work-item desktop bkg-2">
-              <h3>Data Dashboard Healthcare</h3>
-              <p>
-                A daily selection of privately personalized reads; no accounts
-                or sign-ups required. has been the industry's standard
-              </p>
-              <ul class="work-stack">
-                <li class="stack-item">html</li>
-                <li class="stack-item">bootstrap</li>
-                <li class="stack-item">Ruby</li>
-              </ul>
-            </article>
-            <article class="work-item desktop bkg-3">
-              <h3>Website Protfolio</h3>
-              <p>
-                A daily selection of privately personalized reads; no accounts
-                or sign-ups required. has been the industry's standard
-              </p>
-              <ul class="work-stack">
-                <li class="stack-item">html</li>
-                <li class="stack-item">bootstrap</li>
-                <li class="stack-item">Ruby</li>
-              </ul>
-            </article>
-            <article class="work-item desktop bkg-4">
-              <h3>Profesional Art Printing Data More</h3>
-              <p>
-                A daily selection of privately personalized reads; no accounts
-                or sign-ups required. has been the industry's standard
-              </p>
-              <ul class="work-stack">
-                <li class="stack-item">html</li>
-                <li class="stack-item">bootstrap</li>
-                <li class="stack-item">Ruby</li>
-              </ul>
-            </article>
-            <article class="work-item desktop bkg-2">
-              <h3>Data Dashboard Healthcare</h3>
-              <p>
-                A daily selection of privately personalized reads; no accounts
-                or sign-ups required. has been the industry's standard
-              </p>
-              <ul class="work-stack">
-                <li class="stack-item">html</li>
-                <li class="stack-item">bootstrap</li>
-                <li class="stack-item">Ruby</li>
-              </ul>
-            </article>
-            <article class="work-item desktop bkg-3">
-              <h3>Website Protfolio</h3>
-              <p>
-                A daily selection of privately personalized reads; no accounts
-                or sign-ups required. has been the industry's standard
-              </p>
-              <ul class="work-stack">
-                <li class="stack-item">html</li>
-                <li class="stack-item">bootstrap</li>
-                <li class="stack-item">Ruby</li>
-              </ul>
-            </article>
-          </div>
```

Comment:
- [ ] When I click on a project I don't see any pop-up. Please try and fix that.

Reply:
@emmanuellekamwa I just re-cloned now to test out the issue pointed out and the pop-up showed. 
Could please provide more details on what could be wrong?

Did you preview from the modal branch?

Please I need more insight into what the issue is.

Thank you for your review

Reply:
I have tested your project on the live demo link that is on your readme and I cloned your project too but the popup is not still showing.

Reply:
Live demo is served from the main branch, not this feature branch. And to really test out what this pull request, it has to be tested based on the content of the "modal" branch. 

From your clone copy please checkout to modal branch and preview from their!

Reply:
Besides, it is the "See Project" that triggers the pop-up, no the whole card

Reply:
I need to know what is wrong with this first, I cannot just waste a review.
Hope you understand my concern.